lib/clientv3/ippool.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2018 Tigera, Inc. All rights reserved.
+// Copyright (c) 2017-2018,2019 Tigera, Inc. All rights reserved.
```

Comment:
minor thing, but I think we should just say 2017-2019 here

Reply:
@caseydavenport I have updated the copyright.